myfile.txt

```diff
@@ -0,0 +1,3 @@
+Here is a new file!
+
+check it out Jason, you might like it
```

Comment:
This line seems a bit confusing. Maybe we could change the word "it" to "my face'